src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -484,7 +484,7 @@ private void ProcessTraceroute(string targetNameOrAddress)
                                 reply.Status == IPStatus.Success
                                     ? reply.RoundtripTime
                                     : timer.ElapsedMilliseconds,
-                                buffer.Length,
+                                buffer.Length == 0 ? DefaultSendBufferSize : buffer.Length,
```

Comment:
Why do we need to make this change? In case of `BufferSize == 0`, the buffer used to send ping payload is an empty array, so why change the real size to 32 byte here?

Reply:
If we leave it as-is, then the user sees the output objects as using a 0 size buffer even though the actual ICMP request used a default 32 byte buffer.  So this line is to show what happens under the covers.